test/e2e/specs/css.spec.js

```diff
@@ -0,0 +1,67 @@
+var bgColorProp = 'rgba(0, 0, 0, 1)';
+module.exports = {
+  'Basic CSS test' : function (client) {
+       client
+          .url(client.launch_url)
+          .selectTab('CSS')
+          .setCssValue("body{background-color:#000}")
+          .assert.outputBackgroundIs(bgColorProp,client)
```

Comment:
There should be a way to access client from the assertion without passing it explicitly.